### PR TITLE
adjusted css on favorites so cards show correctly and centered

### PR DESCRIPTION
**Pull request by robbiegreiner**
@francylang @adammescher @ameseee @nicktu12

#25 #22 

Beer cards on the favorites page weren't aligning correctly.  Needed to add flex align-items center.

Addresses issues #25 and #22.  
https://github.com/robbiegreiner/brewery-tracker/issues/25
https://github.com/robbiegreiner/brewery-tracker/issues/22

screen shot 2017-11-07 at 10 24 43 am

### CODE REVIEW

**Comment by nicktu12** [comment]
Looks like snapshot tests were updated too

File: src/containers/tests/SearchResultsContainer.test.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ describe('SearchResultsContainer', () => {
   const mockGetBrewery = jest.fn();
   const mockAddFavoriteBeer = jest.fn();
   const mockAddFavoriteBrewery = jest.fn();
-  const store= mockStore(initialState, middlewares);
+  const store = mockStore(initialState, middlewares);
```
Comment (ameseee) [inline src/containers/tests/SearchResultsContainer.test.js:26]: 💯

File: src/styles/index.css [hunk 2]
```diff
@@ -412,7 +412,11 @@
   margin-top: 120px; }
 
 .favorites {
-  margin-top: 120px; }
```
Comment (francylang) [inline src/styles/index.css:415]: Great delete and re-add on that margin.

**Review by francylang — APPROVED** [review]
I approve the work you've done here. Great UI changes.

**Pull request merged**